HeterogeneousCore/AlpakaUtilities/interface/HistoContainer.h

```diff
@@ -0,0 +1,310 @@
+#ifndef AlpakaCore_HistoContainer_h
+#define AlpakaCore_HistoContainer_h
+
+#include <alpaka/alpaka.hpp>
+#include <algorithm>
+#include <cstddef>
+#include <cstdint>
+#include <type_traits>
+
+#include "AtomicPairCounter.h"
+#include "alpakastdAlgorithm.h"
+#include "prefixScan.h"
```

Comment:
Our convention is use full paths to the headers even if they are in the `interface/` directory of the same package (only exceptions are cases where the source and header files are in `src`, `plugins`, or `test`).